apps/callscreen/test/unit/call_screen_test.js

```diff
@@ -830,10 +827,7 @@ suite('call screen', function() {
     });
 
     test('should show the text', function() {
-      assert.equal(
-        statusMessage.querySelector('p').getAttribute('data-l10n-id'),
-        'message'
-      );
+      l10nAssert(statusMessage.querySelector('p'), 'message');
```

Comment:
If you pass a string to l10nAssert it'll check for the data-l10n-id only, see here: https://github.com/mozilla-b2g/gaia/pull/30157/files#diff-32497f9d76f1a7e98c97a5a07ed5898eR23